packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -90,8 +100,8 @@ class UnboundedBehavior extends ExtentScrollBehavior {
     );
   }
 
-  Simulation createSnapScrollSimulation(double startOffset, double endOffset, double velocity) {
-    return _createSnapScrollSimulation(startOffset, endOffset, velocity);
+  Simulation createSnapScrollSimulation(double startOffset, double endOffset, double startVelocity, double endVelocity) {
```

Comment:
I don't understand this method. Why is it an instance method? It doesn't seem to be in any way dependent on the object it's called on? Or am I missing something?

Same with the other methods that call _create*